api/app/controllers/spree/graphql/graphql_controller.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+module Spree
+  module Graphql
+    class GraphqlController < ::Doorkeeper::ApplicationMetalController
+      rescue_from CanCan::AccessDenied, with: :access_denied
+      rescue_from JWT::DecodeError, with: :bad_token
+
+      def create
+        query_string = params[:query]
+        query_variables = ensure_hash(params[:variables])
+        result = GraphqlSchema.execute(query_string, variables: query_variables, context: { spree_current_user: spree_current_user, spree_current_order:  find_spree_current_order})
+        render json: result
+      end
+
+      def login
+        if spree_current_user
+          render json: JwtToken.create_for_user(spree_current_user), status: :ok
```

Comment:
```suggestion
          render json: ::Spree::JwtToken.create_for_user(spree_current_user), status: :ok
```

please use full namespace as in this current place the current namespace is `Spree::Graphql`